examples/basic/variables.tf

```diff
@@ -26,7 +30,7 @@ variable "resource_group" {
   default     = null
 }
 
-variable "resource_tags" {
+variable "tags" {
```

Comment:
change back to `resource_tags` as thats what our test wrapper uses

Reply:
Done.